docs/source/usage/plugins/isaac.rst

```diff
@@ -74,6 +74,20 @@ Beside the functor chains the client allows to setup the weights per source (val
 Furthermore interpolation can be activated.
 However this is quite slow and most of the time not needed for non-iso-surface rendering.
 
+Memory Complexity
+^^^^^^^^^^^^^^^^^
+
+Accelerator
+"""""""""""
+
+locally, a framebuffer with full resolution and 3 color channels and 4 byte is allocated.
```

Comment:
I just took a look. It's an RGBX bytewise framebuffer, so 4 byte per pixel at all.
`locally, a framebuffer with full resolution and 4 byte per pixel is allocated.`